models/spleen_deepedit_annotation/docs/README.md

```diff
@@ -118,10 +118,30 @@ python -m monai.bundle run --config_file "['configs/train.json','configs/evaluat
 
 ####  Execute inference:
 
+
+For automatic inference mode:
+
+
 ```
 python -m monai.bundle run --config_file configs/inference.json
 ```
 
+For interactive segmentation mode, in which the user provides clicks, set the **use_click** flag to true:
+
+
+```
+python -m monai.bundle run --config_file configs/inference.json --use_click true
+```
+
+Clicks should be added to the data dictionary that is passed to the preprocessing transforms. Label names are the keys and their values are the clicks coordinates.
+
+Like the following format:
+
+```
+data_dict = {'background': [], 'spleen': [[I1, J1, K1], [I2, J2, K2]]}
```

Comment:
Hi @diazandr3s @Nic-Ma , for one sample, we need to prepare the data like:
```
{
    "image": "example.nii.gz",
    "background": [],
    "spleen": [[I1, J1, K1], [I2, J2, K2]],
}
```
Where the key names of "background" and "spleen" are determined by `label_name` （https://github.com/Project-MONAI/model-zoo/pull/491/files#diff-a9b28f1d091298054a8eecfef0bce115602c3259d5d2299e0a1cd34da1b5a329L11）

In order to modify the input for `dataset#data` (https://github.com/Project-MONAI/model-zoo/pull/491/files#diff-a9b28f1d091298054a8eecfef0bce115602c3259d5d2299e0a1cd34da1b5a329L109) we should design the format of the "background" and "spleen" values. The following is an example:
```
deepedit_dataset_example/
├── clicksTs
│   ├── background.npy
│   └── spleen.npy
└── imagesTs
    └── image.nii.gz
```
Then, we can modify the config file accordingly. 
I think for this specific bundle, we can use whatever format we want, and we just need to describe it clearly in the readme file (and better enhance the corresponding bundle's unit test with example data)

However, if we'd like to extend this way for more bundles, we need to carefully design a generic format. What do you think?
cc @SachidanandAlle @tangy5 @mingxin-zheng @wyli @KumoLiu

Reply:
do we have to save the clicks to npy file? I feel this will slow down the click-based inference?

Reply:
If consider clicks as a kind of "labels" to be used, I think we should load them from saved files.
According to the source code of `AddGuidanceFromPointsDeepEditd ` (https://github.com/Project-MONAI/MONAI/blob/2daabf917a1e6afbc738d22db602f4859672a90d/monai/apps/deepedit/transforms.py#L680),
The input of this transform has to be a metatensor that contains keys: `image`, `background`, `spleen` and `image_meta_dict`, I'm not sure if not loading from `LoadImaged`, in other parts (of the pipeline), where can we "add" these information. What do you think?